lightning/src/chain/channelmonitor.rs

```diff
@@ -2299,6 +2321,39 @@ impl<Signer: Sign> ChannelMonitorImpl<Signer> {
 	pub fn get_and_clear_pending_events(&mut self) -> Vec<Event> {
 		let mut ret = Vec::new();
 		mem::swap(&mut ret, &mut self.pending_events);
+		#[cfg(test)] // TODO: remove-on-anchors-release
```

Comment:
I wonder if this best practice to abuse Rust compilation flags to feature gate internal LDK feature into completion. I would rather add a new config option `option_anchor_output_v0` and turns it to false by default. It should allow you to exercise new code with our current test framework without us having to wonder if we have not forget a random `cfg(test)` nullifying some critical code paths when the feature is complete. At the same time it provides a modular way for beta users to play with feature. 

What do we think ?